x-pack/legacy/plugins/siem/public/components/timeline/helpers.test.tsx

```diff
@@ -12,60 +12,60 @@ import { mockDataProviders } from './data_providers/mock/mock_data_providers';
 import { buildGlobalQuery, combineQueries } from './helpers';
 import { mockBrowserFields } from '../../containers/source/mock';
 
-const cleanUpKqlQuery = (str: string) => str.replace(/\n/g, '').replace(/\s\s+/g, ' ');
+export const cleanUpKqlQuery = (str: string) => str.replace(/\n/g, '').replace(/\s\s+/g, ' ');
```

Comment:
All references look to be within this file -- is it necessary to export?

Reply:
again left from testing, good eye ;)